velox/expression/ExprUtils.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include "velox/expression/Expr.h"
+
+namespace facebook::velox::expression::utils {
+
+/// Returns true if expr is of type CallTypedExpr and the expr name matches
+/// the passed name. Otherwise, returns false.
+bool isCall(const core::TypedExprPtr& expr, const std::string& name);
+
+/// Utility method to check eligibility for flattening.
```

Comment:
perhaps, clarify what this does, e.g.

/// @return true if all inputs have equivalent types.